notebook/services/kernels/kernelmanager.py

```diff
@@ -463,3 +467,38 @@ def cull_kernel_if_idle(self, kernel_id):
                 self.log.warning("Culling '%s' kernel '%s' (%s) with %d connections due to %s seconds of inactivity.",
                                  kernel.execution_state, kernel.kernel_name, kernel_id, connections, idle_duration)
                 self.shutdown_kernel(kernel_id)
+
+    @gen.coroutine
+    def kernel_ram_monitoring(self):
+        # Arbitrary time for waiting till next metric collection
+        yield gen.sleep(0.2)
```

Comment:
Five passes per second seems a little excessive to me.  I suspect that on highly loaded servers (with lots of kernels running), the set of kernels may not get completed before the next iteration will occur.  If this interval could be configured, that would be nice.  Seems like a default of 1 second would be frequent enough for the normal case (of a couple kernels).